App/notReadyTests/RemoveUserFromGroup.test.tsx

```diff
@@ -0,0 +1,16 @@
+// import { render } from "@testing-library/react";
+// import { BrowserRouter } from "react-router-dom";
+// import RemoveUserFromGroup from "../src/pages/RemoveUserFromGroup";
+
+// describe("Add User to Group Test Case", () => {
```

Comment:
Why is this test commented out?

Reply:
Not able to work on time, will fix next sprint